zhmccli/_cmd_lpar.py

```diff
@@ -308,8 +308,14 @@ def cmd_lpar_load(cmd_ctx, cpc_name, lpar_name, load_address, options):
 
     options = original_options(options)
 
+    load_parameter = ""
+    options = original_options(options)
+    if options['load-parameter']:
+        load_parameter = options['load-parameter']
```

Comment:
This whole logic is too complex. You essential undo what original_options() does. The only change that is needed here based upon the original code is to remove the line:

    options = original_options(options)